From a0b0c2f11062077d9dfbffacfaedb85b48b6941e Mon Sep 17 00:00:00 2001
From: Rich Mattes <richmattes@gmail.com>
Date: Mon, 23 Mar 2015 18:53:12 -0400
Subject: [PATCH 5/7] Don't search for FLUID when searching for FLTK

Stage doesn't need FLUID to be available during build time, but the
CMake FindFLTK package searches for it by default in newer versions.
This patch removes the requirement for FLUID to be present.
---
 CMakeLists.txt | 1 +
 1 file changed, 1 insertion(+)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index cdc5476..39add53 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -102,6 +102,7 @@ ELSE( PNG_FOUND )
    MESSAGE( ${INDENT} "PNG library not detected" )
 ENDIF( PNG_FOUND )
    
+set (FLTK_SKIP_FLUID TRUE)
 find_package( FLTK REQUIRED )
 IF( FLTK_FOUND )
 #  MESSAGE( STATUS ${INDENT} "FLTK_LIBRARIES=${FLTK_LIBRARIES}" )
-- 
2.3.8 (Apple Git-58)

